Automattic/vip-go-mu-plugins
# Remove Jetpack subtrees: 8.8, 8.9, 9.2

## Description

Remove Jetpack subtrees for versions 8.8, 8.9, and 9.2, that are no longer being used by any site.

## Changelog Description

### Remove old unused Jetpack versions

We have removed some Jetpack versions (8.8, 8.9, 9.2) from `vip-go-mu-plugins`, that were no longer in use by any VIP Go site. Every included version takes space in the repository, so this is a needed maintenance task.

## Checklist


## Steps to Test

1. Check out PR.
1. Go to the Jetpack dashboard in `wp-admin`. It should still be 9.3.1 (the current default version)
1. Add the `VIP_JETPACK_PINNED_VERSION` define set to 8.8 (a removed version). The loaded version in the dashboard should still be 9.3.1 (there's a fallback for non-existing versions)
